swarm/vulcan.1650566317.yml

```diff
@@ -63,7 +63,7 @@ services:
       - |
         set -e
         export METRICS_PW="$$(cat /run/secrets/metrics-pw)"
-        exec /bin/deploy.sh migrate 'docker pull etnaagent/archimedes:production' 'docker pull etnaagent/archimedes-node:production' -- /app/bin/puma.sh
+        exec /bin/deploy.sh migrate 'docker pull etnaagent/archimedes:production' 'docker pull etnaagent/archimedes-node:production' 'docker pull etnaagent/vulcan:production' -- /app/bin/puma.sh
```

Comment:
Also need to make sure that the `etnaagent/vulcan:production` image with tag are up-to-date on the production system, so we can pin the dependency.